apache/superset
# fix: Validate jinja rendered query

### SUMMARY
The query wasn't validated with the rendered Jinja template. This fix passes the rendered query down to the validator.

### TESTING INSTRUCTIONS
1. Create a query in SQLlab using Jinja params for the schema
2. Restrict access to the schema
3. Verify that the user with the restricted access cannot query the schema when using Jinja params

### ADDITIONAL INFORMATION
